uber/configspec.ini

```diff
@@ -332,6 +332,12 @@ prereg_takedown = string(default="2016-01-18")
 # able to edit them after this happens.
 uber_takedown = string(default="2016-01-21")
 
+# On the last day of an event, we usually let people in for free when it's so close to
+# the end of the day that it would be silly to charge someone for e.g. an hour or two.
+# Badges registered through the at-the-door registration form after this time will
```

Comment:
Isn't this a terrible, terrible thing for our records? We should be marking them as 'need not pay' right?

Otherwise someone could come in and claim they paid and want a refund, and the system would show that they paid, and we'd be all atizzy trying to figure out what happened.